### ISSUE TITLE
mad respect

### ISSUE THREAD

**Issue by wictorianx** [zackmawaldi/Chemical-Equation-Balancer#1]
dude how does this even work! mad respect!

**Comment by zackmawaldi** [comment]
Haha thanks! The code is pretty spaghetti and hard to follow: this was one of my first coding projects.
It works by putting the equation into a linear combination, then finding Reduced Row-Echelon Form (RREF) of it.

Here's a video example:
https://www.youtube.com/watch?v=yCxDAj87W8M

Glad that you liked it!!
